compiler/rustc_middle/src/mir/interpret/allocation/provenance_map.rs

```diff
@@ -195,6 +195,22 @@ impl<Prov: Provenance> ProvenanceMap<Prov> {
 
         Ok(())
     }
+    pub fn write_wildcards(&mut self, alloc_size: usize) {
```

Comment:
Please add a doc comment (and leave an empty line between the previous function and this one)

Reply:
I did this for now - does this work well enough?
Or should I specify conditions for the assert instead, possibly make the `Prov::WILDCARD.unwrap()` explicit there as well?
```rust
    /// Overwrites all provenance in the allocation with wildcard provenance.
    /// 
    /// Provided for usage in Miri and panics otherwise.
    pub fn write_wildcards(&mut self, alloc_size: usize) {
        assert!(
            Prov::OFFSET_IS_ADDR,
            "writing wildcard provenance is not supported when `OFFSET_IS_ADDR` is false"
        );
        let wildcard = Prov::WILDCARD.unwrap();

        // ...
```

Reply:
That comment works just fine. :)